### DISCUSSION TITLE
stb meaning

### DISCUSSION THREAD

**Discussion by adammaj1** [nothings/stb#1401] [Q&A]
Hi
What is the meaning of stb in stb-style ? Is it acronyme ? I know that it is stb single-file public domain libraries for C/C++. 

TIA

Adam

**Comment by visualdoj** [comment]
See FAQ section in the readme of the repository: https://github.com/nothings/stb#why-stb-is-this-something-to-do-with-set-top-boxes

> Why "stb"? Is this something to do with Set-Top Boxes?
> 
> No, they are just the initials for my name, Sean T. Barrett. This was not chosen out of egomania, but as a moderately sane way of namespacing the filenames and source function names.

**Comment by adammaj1** [comment]
Ok. Thx. See also [this SO answer](https://stackoverflow.com/questions/389827/namespaces-in-c/74276997#74276997l)
